.pre-commit-config.yaml

```diff
@@ -20,6 +20,15 @@ repos:
     -   id: check-yaml
 
 
+-   repo: https://github.com/pycqa/isort
+    rev: 5.9.3
+    hooks:
+    - id: isort
+      files: >-
+         (?x)^(
+            aiida/common/.*py|
+         )$
```

Comment:
I can appreciate that doing everything now would cause merge conflicts with open PRs, but I fear that in this way it is never going to get done. If we are going to add this, I prefer we bite the bullet. Since it only concerns imports, the merge conflicts should not be complicated.

Reply:
Sure ting, more lines of code attributed to me lol